shared/src/business/useCases/courtIssuedDocument/fileAndServeCourtIssuedDocumentInteractor.test.ts

```diff
@@ -344,6 +356,33 @@ describe('fileAndServeCourtIssuedDocumentInteractor', () => {
     expect(expectedDocketEntry.isOnDocketRecord).toBe(true);
   });
 
+  it('should set the filingDate to be today even if it was already set on the past', async () => {
```

Comment:
what's the point of this unit test when there's an integration test covering the bug?

Reply:
i could totally be missing something.